extension/src/vscode/resourcePicker.ts

```diff
@@ -34,3 +28,26 @@ export const pickResources = (title: Title): Thenable<Uri[] | undefined> => {
     title
   })
 }
+
+export const pickFiles = async (
```

Comment:
Had to create a new picker since `pickResources` allows folder selection.

Reply:
Maybe `pickResources` could take two more parameters `canSelectFolders` defaulting to `true` and `canSelectMany` defaulting to `true` also. That way you could remove the duplication between `pickFile`, `pickResources` and `pickFiles`.

Reply:
Done. Good idea :)